src/main/kotlin/racingcar/model/Car.kt

```diff
@@ -0,0 +1,15 @@
+package racingcar.model
+
+import camp.nextstep.edu.missionutils.Randoms
+import racingcar.util.Constant.CAR_MOVEMENT_CONDITION
+
+class Car(val name: String) {
+    var position: Int = 0
+        private set
+
+    fun move() {
+        if (Randoms.pickNumberInRange(0, 9) >= CAR_MOVEMENT_CONDITION) {
+            position++
+        }
+    }
+}
```

Comment:
I hope you can try "data class" next time. In Kotlin, you can define a data class using `data class`. This is one of the reasons why I like Kotlin.💜

```
// java
public class Person {
    private String name;
    private int age;

    public Person(String name, int age) {
        this.name = name;
        this.age = age;
    }

    public String getName() {
        return name;
    }

    public int getAge() {
        return age;
    }

    @Override
    public String toString() {
        return "Person{" +
                "name='" + name + '\'' +
                ", age=" + age +
                '}';
    }
}


// Kotlin data class
data class Person(val name: String, val age: Int)
```

Reply:
Thanks to your suggestion, this week I applied a data class. I agreed that the data class is super handy for reducing boilerplate and improving readability.
Thanks for the great tip! 🙌